src/index.ts

```diff
@@ -240,20 +276,32 @@ export class MondayRedisService extends BaseConnector {
     )
   }
 
-  // Increase (or descrease) the value for one column of a specific
+  // Increase (or decrease) the value for one column of a specific
   // item. The column must have a numeric value and must not be
   // encrypted.
   //
   // The new value is atomically written to the Redis mirror, and will
   // eventually be synchronized back to Monday. This assures updates
   // are atomic and reduces load on Monday, as multiple updates to the
-  // same column are consolidated befor Monday is updated.
+  // same column are consolidated before Monday is updated.
   //
   // @param itemId id of the Monday item to be updated
   // @param title title of the Monday column to be updated
   // @param incr numeric increment (negative to decrement)
   //
   public async incrColumnValue(itemId: string, title: string, incr = 1) {
+    const currentValue = await this.redis.hget(this.keyForItem(itemId), title)
```

Comment:
This code is probably redundant - see my comments above.

Reply:
correct